VERSION.cmake

```diff
@@ -7,6 +7,8 @@ set(MIRALL_VERSION_PATCH 50)
 set(MIRALL_VERSION_YEAR  2025)
 set(MIRALL_SOVERSION     0)
 set(MIRALL_PREVERSION_HUMAN "3.17.0 alpha")  # For preversions where PATCH>=50. Use version + alpha, rc1, rc2, etc.
+set(WINDOWS_INTEGRATION_VERSION 4)
+set(WINDOWS_INTEGRATION_SOVERSION 4)
```

Comment:
where does the `4` come from? (it surely can't be `rand()` ;-)) 

`NCExt/Version.h` defines the [`NCEXT_BUILD_NUM` constant with `47`](https://github.com/nextcloud/desktop/blob/3bdc2ebad7f5f99898b4d0a081097d40078d96ed/shell_integration/windows/NCUtil/Version.h#L3-L5) that is part of the `NCEXT_VERSION` constant

and how does setting these as target properties differ from having it be defined in the `.rc` files?  e.g. https://github.com/nextcloud/desktop/blob/3bdc2ebad7f5f99898b4d0a081097d40078d96ed/shell_integration/windows/NCOverlays/NCOverlay.rc#L67-L68

Reply:
fwiw I didn't see anything different on the resulting DLLs with these two settings

Reply:
yes
that is a leftover
I can double check but I probably have a stash upgrading the DLL embedded version number from a more centralized working place

Reply:
adding the version(s) of the shell integration to the centralised _VERSION.cmake_ file is a good idea :>

would be cool if you still have the stash somewhere.  otherwise I can take care of that as well